frontend/packages/shared/src/hooks/queries/useDataModelsJsonQuery.ts

```diff
@@ -0,0 +1,17 @@
+import type { UseQueryResult } from '@tanstack/react-query';
+import { useQuery } from '@tanstack/react-query';
+import { useServicesContext } from 'app-shared/contexts/ServicesContext';
+import { QueryKey } from 'app-shared/types/QueryKey';
+import type { DataModelMetadataJson } from 'app-shared/types/DataModelMetadata';
+import type { AxiosError } from 'axios';
```

Comment:
I think we can use `Error` instead of `AxiosError`. Then we do not need the third-party import from `axios` as we try to go away from it 😄